examples/complete/components/spacelift-automation/main.tf

```diff
@@ -0,0 +1,12 @@
+module "automation" {
+  source = "../../../../"
```

Comment:
I'm confused about why we need the root module at `examples/complete` and this root module. Are they not the same? Do we need the one at the root of `complete/` or is it a one or the other thing? Mind sharing your thoughts here?

Reply:
At first, I had the root module at `examples/complete`. It was used to create a simple example to spin up `random-pet` stack. Then we discussed that it would be useful to show that `spacelift-automation` can manage itself, so I added its configuration to the expected path - `./examples/complete/components/spacelift-automation/`. That's why we have them both 😕 

I agree that it's excessive, but we need to cut the `examples/complete` root module in this case. Does it make sense?

Reply:
@gberenice I think it makes sense if we put a README in `examples/complete` that provides info on "Go to `examples/complete/spacelift-automation` to run that root module, it will manage itself like so...". We can do that correct?